community/sample-chart/README.md

```diff
@@ -118,3 +118,6 @@ Alternatively, a YAML file that specifies the values for the parameters can be p
 ## Documentation
 * Can have as many supporting links as necessary for this specific workload however don't overload the consumer with unnecessary information.
 * Can be links to special procedures in the knowledge center.
+
+## Support
+* This section should provide details and/or links to where users can get support for urgent issues with the product, get help, or submit issues.
```

Comment:
I think we can keep this yet more general unless we expect them to have at least one of the support items listed:  
```suggestion
* This section should either link to or describe what support is available for the product.  
```